### ISSUE TITLE
Section 508 502.3.1: (devenv/tab item 'Microsoft.ML.ModelBuilder.ImageFolderModel') The Name property must not include its class name.

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#951]
The following accessibility issue needs investigation.

**App:** devenv

**Element path:** tab item 'Microsoft.ML.ModelBuilder.ImageFolderModel'

**Issue Details:** The Name property must not include its class name. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide a UI Automation Name property for the element that:
 · Concisely identifies the element, AND
 · Does not include the element's class name (such as Microsoft.*.* or Windows.*.*).

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

![image](https://user-images.githubusercontent.com/9122518/90283809-76be3a80-de36-11ea-9cc1-91d9e7663642.png)

**Comment by beccamc** [comment]
This should be closed by other accessibility bugs.
